debian/changelog

```diff
@@ -1,5 +1,11 @@
-datalad (0.0.3-1) UNRELEASED; urgency=low
+datalad (0.0.3+git321-gee70b90.1-1) UNRELEASED; urgency=low
```

Comment:
just not yet ;-)  I would prefer 1.0 be somewhat feature complete, and while we are getting there still allow us to break things radically

Actually I got to dislike this rush for the leading version number growth in some projects (firefox, even ipython) since you loose ability to see if something broke (e.g. public API).